framework/include/st_things/st_things.h

```diff
@@ -49,43 +53,51 @@ extern "C" {
  *            If easy-setup is not done, app can either wait for the user interaction before starting the things stack or
  *            start the things stack directly without waiting for any events(This case is for those things which doesn't
  *            support input capability and for all other unknown cases).
+ *            To use a new json file after initialization, stack should be deinitialized
+ *            and stopped(if its started already).
  * @param[in] json_path Path to Json file which defines a thing. Definition includes the device information,
  *                                       resources and their properties, configuration info for connectivity and easy-setup, etc.
  * @param[out] easysetup_complete Indicates whether easysetup is completed already or not.
  * @return @c 0 on success, otherwise a negative error value
  * @retval #ST_THINGS_ERROR_NONE Successful
  * @retval #ST_THINGS_ERROR_INVALID_PARAMETER Invalid parameter
  * @retval #ST_THINGS_ERROR_OPERATION_FAILED Operation failed
- * @since Tizen RT v1.1
+ * @retval #ST_THINGS_ERROR_STACK_ALREADY_INITIALIZED Stack already initialized.
+ *         To initialize again, stack should be deinitilized first by calling st_things_deinitialize().
+ * @retval #ST_THINGS_ERROR_STACK_RUNNING Stack is currently running.
+ *         To initialize again, stack should be stopped first by calling st_things_stop()
+ *         and then deinitialized by calling st_things_deinitialize().
  */
-int st_things_initialize(const char *json_path, bool *easysetup_complete);
+	int st_things_initialize(const char *json_path, bool * easysetup_complete);
```

Comment:
revert this line. Previous codes are right. same for all of prototype in this file

Reply:
modified